src/views/FlawSearchView.vue

```diff
@@ -15,18 +15,15 @@ const searchStore = useSearchStore();
 const { addToast } = useToastStore();
 const tableFilters = ref<Record<string, string>>({});
 
-defineEmits<{
-  'issues:load': [any[]];
-}>();
-
 const params = computed(() => {
   const searchParams = getSearchParams();
-  if (searchParams.order) {
-    searchParams.order += ',' + tableFilters.value.order;
-  }
+  const order = [searchParams.order, tableFilters.value.order]
+    .filter(Boolean).join(',');
+
```

Comment:
I changed this because with the change implemented in #396 `tableFilters.value` can be `null` so if `searchParams.order` was defined, resulted in `created_dt,null`